src/components/NcAppNavigationNew/NcAppNavigationNew.vue

```diff
@@ -78,6 +79,10 @@ export default {
 			type: String,
 			required: true,
 		},
+		type: {
+			type: String,
+			default: 'primary',
```

Comment:
```suggestion
			default: 'primary',
			validator(value) {
				return ['primary', 'secondary', 'tertiary'].indexOf(value) !== -1
			},
```

This would restrict the `type` to the useful ones for the app menu, but it also would add the supported values to the documentation.